### PR TITLE
Updates troubleshooting page

### PR DESCRIPTION
**Pull request by ysims**
This was a very old branch with these two additions. I also updated the list of members to get help from.

### CODE REVIEW

File: src/book/03-guides/04-general/02-troubleshooting.mdx [hunk 1]
```diff
@@ -10,12 +10,14 @@ slug: /guides/general/troubleshooting
...
-- **NUsight**: Brendan Annable, Josephus Paye II, Trent Houliston
-- **Mathematics**: Ysobel Sims, Timothy Mullen
+- **Hardware**: Taylor Young
+- **Toolchain and build**: Kip Hamiltons
+- **NUClear**: Cameron Murtagh
+- **NUsight**: Josephus Paye II, Brendan Annable
+- **Machine Learning**: Joel Wong
+- **Mathematics**: Kip Hamiltons, Cameron Murtagh, Taylor Young, Ysobel Sims
+
+Team mentors Alex Biddulph and Trent Houliston can help with most parts of the system if this list does not help.
```
Comment (JosephusPaye) [inline src/book/03-guides/04-general/02-troubleshooting.mdx:20]: Might be missed if only looking at the list above

```suggestion
Team mentors **Alex Biddulph** and **Trent Houliston** can help with most parts of the system if this list does not help.
```

File: src/book/03-guides/04-general/02-troubleshooting.mdx [hunk 2]
```diff
@@ -61,9 +63,25 @@ find . -type f -name CMakeCache.txt -delete && find . -type d -name CMakeFiles -
 If the problem persists, try purging the build volume by running `./b configure --clean`.
 Note that purging the build volume means you'll have to rerun `./b build` to use built code.
 
+### Not enough memory in Docker? (Mac OS)
```
Comment (JosephusPaye) [inline src/book/03-guides/04-general/02-troubleshooting.mdx:66]:
```suggestion
### Not enough memory in Docker? (macOS)
```

File: src/book/03-guides/04-general/02-troubleshooting.mdx [hunk 3]
```diff
@@ -61,9 +63,25 @@ find . -type f -name CMakeCache.txt -delete && find . -type d -name CMakeFiles -
...
+
+By default the docker install will allocate 2GB of memory to docker images. When building the images this is insufficent for some of the libraries and will result in an error. In the menu bar go to advanced settings and increase the memory available.
```
Comment (JosephusPaye) [inline src/book/03-guides/04-general/02-troubleshooting.mdx:68]:
```suggestion
By default Docker Desktop will allocate 2GB of memory to use for building images. When building the NUbots image, this is insufficient and will result in an error. To fix this, open the Docker menu in the menu bar, go to advanced settings, and increase the memory available.
```

File: src/book/03-guides/04-general/02-troubleshooting.mdx [hunk 4]
```diff
@@ -61,9 +63,25 @@ find . -type f -name CMakeCache.txt -delete && find . -type d -name CMakeFiles -
...
 ## NUsight
 
-TODO
+### Multiple Docker robots in NUsight?
+
+This means someone else is also running a Docker robot. You can modify the name of your Docker robot by doing the following.
+
+```bash
+nano module/support/configuration/NetworkConfiguration/data/config/NetworkConfiguration.yaml
+```
```
Comment (JosephusPaye) [inline src/book/03-guides/04-general/02-troubleshooting.mdx:78]:
```suggestion

1. Open the `NetworkConfiguration.yaml` file in a text editor (here we are using nano on the command line):

   ```bash
   nano module/support/configuration/NetworkConfiguration/data/config/NetworkConfiguration.yaml
   ```
```

File: src/book/03-guides/04-general/02-troubleshooting.mdx [hunk 5]
```diff
@@ -61,9 +63,25 @@ find . -type f -name CMakeCache.txt -delete && find . -type d -name CMakeFiles -
...
+
+Write in the name you want to use in the `name: ""` section and press `CTRL+S` to save the file and then `CTRL+X` to exit.
```
Comment (JosephusPaye) [inline src/book/03-guides/04-general/02-troubleshooting.mdx:80]:
```suggestion
2. Update the `name` field with the name you want to use and press <kbd>Ctrl</kbd>+<kbd>S</kbd> to save the file and then <kbd>Ctrl</kbd>+<kbd>X</kbd> to exit.
```

File: src/book/03-guides/04-general/02-troubleshooting.mdx [hunk 6]
```diff
@@ -61,9 +63,25 @@ find . -type f -name CMakeCache.txt -delete && find . -type d -name CMakeFiles -
...
+
+Rebuild using `./b build` for the changes to take effect.
```
Comment (JosephusPaye) [inline src/book/03-guides/04-general/02-troubleshooting.mdx:82]:
```suggestion
3. Rebuild using `./b build` for the changes to take effect.
```

File: src/book/03-guides/04-general/02-troubleshooting.mdx [hunk 7]
```diff
@@ -61,9 +63,25 @@ find . -type f -name CMakeCache.txt -delete && find . -type d -name CMakeFiles -
...
+
+In NUsight, toggle off the robots you do not want to see by clicking the robot icon in the top right of the NUsight GUI and toggling the slider next to the name of the robot you want to toggle off.
```
Comment (JosephusPaye) [inline src/book/03-guides/04-general/02-troubleshooting.mdx:84]:
```suggestion
4. In NUsight, toggle off the robots you do not want to see by clicking the robot icon at the top right of the NUsight GUI and toggling the slider next to the name of every other robot except the one you set in step 2.
```

**Review by JosephusPaye — CHANGES_REQUESTED** [review]
Some suggestions

File: src/book/03-guides/04-general/02-troubleshooting.mdx (hunk 4 above)
Comment (ysims) [reply src/book/03-guides/04-general/02-troubleshooting.mdx:78]:
```suggestion

1. Open the `NetworkConfiguration.yaml` file in a text editor (here we are using nano on the command line):
   ```bash
   nano module/support/configuration/NetworkConfiguration/data/config/NetworkConfiguration.yaml
   ```
```

Comment (ysims) [reply src/book/03-guides/04-general/02-troubleshooting.mdx:78]: Looks like the ending ``` got cut off, good job GitHub 😆 
I'll commit and add it in manually.

**Review by JosephusPaye — APPROVED** [review]
LGTM!

**Pull request merged**